src/Data/Bool/Properties.agda

```diff
@@ -29,7 +29,7 @@ open import Relation.Binary.Definitions
 open import Relation.Binary.PropositionalEquality.Core
 open import Relation.Binary.PropositionalEquality.Properties
 open import Relation.Nullary.Reflects using (ofʸ; ofⁿ)
-open import Relation.Nullary.Decidable.Core using (True; does; proof; yes; no)
+open import Relation.Nullary.Decidable.Core using (True; does; proof; yes; no; fromWitness)
```

Comment:
Similarly, `does; proof` in the import list here.
And as I have observed elsewhere, it's not clear to me, given the redefinition of `T?-diag` in terms of `fromWitness` that `T?-diag` as a name shouldn't also be deprecated in favour of the latter. Or even removed: is it used anywhere by anyone?

Reply:
Removed them.